lib/registrant.rb

```diff
@@ -0,0 +1,20 @@
+class Registrant
+  attr_reader :name, :age
+  attr_accessor :license_data, :permit
```

Comment:
I don't think you'll need to list `:permit` as an attr_accessor here. If you look throughout you're code, you never need to call `.permit` and assign it to a new value in another class. I actually think you could get rid of this altogether (no need for `:permit` as a reader or an accessor), because you can check the status of the permit by calling `permit?`, and you can change the status of the permit by calling `earn_permit`.